src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -176,9 +181,6 @@ private static void EmitRegexDerivedTypeRunnerFactory(IndentedTextWriter writer,
             writer.WriteLine($"    /// <summary>Provides the runner that contains the custom logic implementing the specified regular expression.</summary>");
             writer.WriteLine($"    private sealed class Runner : RegexRunner");
             writer.WriteLine($"    {{");
-            writer.WriteLine($"        // Description:");
```

Comment:
Just my 2 cents here, but could it still be valuable to have the description inside the implemented class as well? Main scenario I'm thinking of is when you have an app with several regex source-generated types and you want to look or debug into one of the implementations. it might be handy to have the whole tree description close to the implementation as opposed to have it all the way up in the partial method declaration. Don't feel too strongly about it though so it's fine if we decide to keep as is and then listen for feedback if we get any around this.

Reply:
> Don't feel too strongly about it though so it's fine if we decide to keep as is and then listen for feedback if we get any around this.

I'd prefer this for now.  We already render the same comments on each block of code in the implementation, so adding it again would be yet a third output of it.